lib/tasks/maintenance.rake

```diff
@@ -225,6 +225,40 @@ namespace :maintenance do
     end
   end
 
+  task migration_contract_duration: :environment do
+    puts "Did you set duration boolean on all contract_type ? (y/N)"
+    response = STDIN.gets.chomp.downcase
+    next unless %w[yes y].include?(response)
+    puts "Migrating..."
+
+    # Create missing contract duration
+    (1..11).each do |i|
+      ContractDuration.find_or_create_by!(name: "#{i} mois")
+      ContractDuration.find_or_create_by!(name: "#{i} mois renouvelable")
+    end
+    ContractDuration.find_or_create_by!(name: "1 an")
+    ContractDuration.find_or_create_by!(name: "1 an renouvelable")
+    (2..5).each do |i|
+      ContractDuration.find_or_create_by!(name: "#{i} années")
```

Comment:
make it simple here: replace "année" by "an". Therefore the loop should from 1 to 5.

Reply:
`an` also take an `s`, no ?

Reply:
But it should be `an/ans` or `année/années`

Reply:
yep. Or use `pluralize` :)

Reply:
`pluralize` with the `count` option.